sdk/lib/opentelemetry/sdk/configurator.rb

```diff
@@ -27,7 +27,7 @@ def initialize
         @text_map_injectors = nil
         @span_processors = []
         @use_mode = USE_MODE_UNSPECIFIED
-        @resource = Resources::Resource.telemetry_sdk
+        @resource = Resources::Resource.telemetry_sdk.merge(Resources::Resource.process)
```

Comment:
This is comment that is probably splitting hairs but, is there any reason to put this here versus in inside the [configure block](https://github.com/open-telemetry/opentelemetry-ruby/pull/557/files#diff-e6a05d2b9d8d72a4c1ab589e75a9aa34e484e073af6443d428929c0c554f93eaR116-R125)?

Reply:
🤷‍♂️